beacon_node/eth1/src/service.rs

```diff
@@ -140,7 +149,7 @@ async fn endpoint_state(
         warn!(
             log,
             "Error connecting to eth1 node endpoint";
-            "endpoint" => endpoint,
+            "endpoint" => endpoint.to_string(),
```

Comment:
We should use the magic `%endpoint` syntax for the `Display` implementation (there are a few in this file)

```suggestion
            "endpoint" => %endpoint,
```

Reply:
Ahh yes, I had forgotten about %. Thanks!